build-scripts/build-bindings.sh

```diff
@@ -4,9 +4,7 @@
 # Important note: 
 # Run from root directory
 
-# Parse package version from the project
-meson setup builddir
-VERSION="$(meson introspect --projectinfo builddir | jq -r '.version')"
+VERSION_SCRIPT=$(dirname "$(readlink -f "$0")")/version.sh
```

Comment:
Is there a reason to replace `VERSION` with `VERSION_SCRIPT`? It seems to me less clear ...

Reply:
No real reason. I changed it to directly retrieve the version. 
PTAL @mwperina